src/guide/advanced/authentication.md

```diff
@@ -200,16 +208,186 @@ To request a protected route, you will need to add the following header:
 Authorization: Bearer <YOUR_TOKEN>
 ```
 
+##### Optional authentication
+
 This provider supports the `Optional` flag, which defines if the authenticator allows requests that don't provide credentials. Handlers should therefore check if `request.User` is not `nil` before accessing it.
 
 ``` go
 authenticator := auth.Middleware(&model.User{}, &auth.JWTAuthenticator{Optional: true})
 router.Middleware(authenticator)
 ```
 
+##### Custom ID claim name
```

Comment:
These titles are very small in the theme in comparison of h4

Reply:
They have `1rem` size, have bold font and have some margin. I think they look large enough.